docs/oss/security/FasterXML/jackson-dataformat-xml.md

```diff
@@ -93,18 +118,18 @@ Here is how the rating was calculated:
       Value:........0.50  out of 10.00
       Confidence:...10.00 out of 10.00
       Based on:...2 features:
-          Number of stars for a GitHub repository:......396
-          Number of watchers for a GitHub repository:...31
+          Number of stars for a GitHub repository:......397
+          Number of watchers for a GitHub repository:...32
 
       Sub-score:....Vulnerability lifetime
       Importance:...Low (weight 0.23 out of 1.00)
-      Value:........0.00  out of 10.00
+      Value:........6.25  out of 10.00
       Confidence:...10.00 out of 10.00
       Based on:...2 features:
-          Info about vulnerabilities:...5 vulnerabilities
+          Info about vulnerabilities:...2 vulnerabilities
```

Comment:
`FALSE POSITIVE` cases fixed here as well.

Reply:
Yes, now it looks correct

https://www.cvedetails.com/vulnerability-list/vendor_id-15866/product_id-61565/Fasterxml-Jackson-dataformat-xml.html